src/MCPClient/requirements/base.txt

```diff
@@ -0,0 +1,11 @@
+bagit==1.5.2
+Django>=1.7,<1.8
+django-annoying==0.7.7
+django-autoslug==1.7.1
+django-mysqlpool==0.1-9
+django-extensions==1.1.1
+mysqlclient==1.3.7
```

Comment:
`mysqlclient` is a fork of the `MySQL-python` library, not the actual MySQL command-line tool, if that's what you meant. I don't really think that the Ubuntu package `mysql-client` is a dependency whereas `rsync` is a dependency because it's used by one of the client scripts. Thoughts?